### PR TITLE
Documentation for running arm64 builds

### PR DESCRIPTION
**Pull request by docbobo**
Attempt to clarify documentation around arm64. Let me know, what you think

### CODE REVIEW

File: docs/docker_quickstart.md [hunk 1]
```diff
@@ -67,29 +67,33 @@ Create a new directory and place the [docker-compose file](https://raw.githubuse
...
     cd ft_userdata/
-    # Download the docker-compose file from the repository
-    curl https://raw.githubusercontent.com/freqtrade/freqtrade/stable/docker-compose.yml -o docker-compose.yml
 
-    # Edit the compose file, uncomment the "build" step and use "./docker/Dockerfile.aarch64"
-    # Also, change the image name to something of your liking
+    # arm64 images are not yer provided via Docker Hub and need to be build locally first. Depending on the device,
+    # this may take a few minutes (Apple M1) or up to two hours (Raspberry Pi)
+    git clone  https://github.com/freqtrade/freqtrade.git
+    docker build -f ./freqtrade/docker/Dockerfile.aarch64 -t freqtradeorg/freqtrade:develop_arm64 freqtrade
```
Comment (xmatthias) [inline docs/docker_quickstart.md]: Don't recommend doing it this way - but instead use docker-compose to build the image.

The compose-file will need to be edited anyway (the image name is now different) - so using `docker-compose build` will be the simpler command (and will also be simpler in update-scenarios).

File: docs/docker_quickstart.md [hunk 2]
```diff
@@ -67,29 +67,33 @@ Create a new directory and place the [docker-compose file](https://raw.githubuse
...
 
     ``` bash
     mkdir ft_userdata
     cd ft_userdata/
-    # Download the docker-compose file from the repository
-    curl https://raw.githubusercontent.com/freqtrade/freqtrade/stable/docker-compose.yml -o docker-compose.yml
 
-    # Edit the compose file, uncomment the "build" step and use "./docker/Dockerfile.aarch64"
-    # Also, change the image name to something of your liking
+    # arm64 images are not yer provided via Docker Hub and need to be build locally first. Depending on the device,
```
Comment (xmatthias) [inline docs/docker_quickstart.md]:
```suggestion
    # arm64 images are not yet provided via Docker Hub and need to be build locally first. Depending on the device,
```

File: docs/docker_quickstart.md [hunk 3]
```diff
@@ -67,29 +67,33 @@ Create a new directory and place the [docker-compose file](https://raw.githubuse
...
+    # this may take a few minutes (Apple M1) or up to two hours (Raspberry Pi)
```
Comment (xmatthias) [inline docs/docker_quickstart.md]: i'd keep it at "multiple hours" - as this will not only depend on the raspberry, but also on the network speed.

**Review by Rada8732 — APPROVED** [review]

File: docs/docker_quickstart.md (hunk 1 above)
Comment (docbobo) [reply docs/docker_quickstart.md]: Fair enough. Personally, I think that gets a little bit more tedious when you are trying to build anything on top of the other Dockerfiles inside `./docker`. But if you prefer it that way, I'll happily change that.

Comment (xmatthias) [reply docs/docker_quickstart.md]: usually, mixing docker + docker-compose is quite confusing to people not used to docker (at least that's my experience from trying to explain it to some colleagues at work) ...

**Review by xmatthias — APPROVED** [review]
:+1: 
Great catch to notice that the docs actually don't work if you're not checked out!

**Pull request merged**